dev-cpp/gsl/gsl-0_pre20171105.ebuild

```diff
@@ -18,6 +18,8 @@ IUSE="test"
 RDEPEND=""
 DEPEND="test? ( >=dev-cpp/catch-1.11.0 )"
 
+PATCHES="${FILESDIR}/${P}-use_system_catch-636828.patch"
```

Comment:
PATCHES is usually specified as an array, so I suggest you change this to:
```
PATCHES=(
"${FILESDIR}/${P}-use_system_catch-636828.patch"
)
```

Reply:
Okay, I updated my patch accordingly.